app/views/ajax/home.html.erb

```diff
@@ -50,5 +138,90 @@ document.addEventListener('DOMContentLoaded', (event) => {
     console.log("sending data:",dataToSend)
     eventRequest.send(dataToSend);
   });
+
+
+  const addItemButton = document.getElementById('add-item');
+  const item_name = document.getElementById('item-name');
+  const item_todo_id = document.getElementById('item-todo-id')
+  //const checkbox = document.getElementById("done_check").checked;
+  const checkbox = document.querySelector('input[value="Done"]');
+  addItemButton.addEventListener('click', (event) => {
+    var eventRequest = new XMLHttpRequest();
+    eventRequest.onreadystatechange = function () {
+      if(eventRequest.readyState === XMLHttpRequest.DONE) {
+        if  (eventRequest.status === 201) {     // this is the expected return code for a POST to create a todo
+          //resultsDiv.textContent=eventRequest.responseText; // this will be the todo that is created
+          alert("Item Successfully Added to ToDo list")
```

Comment:
It might be nice to actually show the item that was added, including the id number and timestamps, instead of just the alert.